model/src/service/path.py

```diff
@@ -30,10 +30,9 @@ def process_configfile():
 
 config_data = process_configfile()
 
-REDIS_HOST = config_data["Hostname"]
 HOP = config_data["Hop"]
 # TYPE = config_data["Type"]
 INFLUXDB_DATABASE = "latency"
 INFLUXDB_HOST = "influxdb"
 INFLUXDB_PASSWORD = "root"
-INFLUXDB_USERNAME = "root"
+INFLUXDB_USERNAME = "root"
```

Comment:
Do we have any `Hop` inside our config data? Also about the InfluxDB, we don't have it anymore.

Reply:
we have "Hop" in the previous version of the configmap data whose format was something like: {"Hop":{"1":["service-3"],"2":["service-3"],"3":["service-3"],"4":null,"5":null,"6":null,"7":null,"8":null},"Hostname":"service-1"}. This will be updated once we fix the newly created issue #28.

For the InfluxDB, after discussing we won't make use of it anymore... so, I will update the code accordingly.